roles/test_dashboards_loaded/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
+# tasks file for roles/test_dashboards_loaded
+- name: "Subscribe to the Grafana Operator"
+  ansible.builtin.shell:
+    cmd: |
+      oc apply -f - <<EOF
+      apiVersion: operators.coreos.com/v1alpha1
+      kind: Subscription
+      metadata:
+        name: grafana-operator
+        namespace: service-telemetry
+      spec:
+        channel: v4
+        installPlanApproval: Automatic
+        name: grafana-operator
+        source: community-operators
+        sourceNamespace: openshift-marketplace
+      EOF
+
+  changed_when: false
+  register: cmd_output
+  failed_when: cmd_output.rc != 0
+
+- name: "Manual Approve installplan"
```

Comment:
it didn't work in my env without it so I wasn't sure if it may cause a problem, I know its an extra step that may not be needed but it takes seconds in running and not causing any conflicts , but I can try to test it again on another fresh env and check .